msal4j-sdk/src/main/java/com/microsoft/aad/msal4j/ManagedIdentityRequest.java

```diff
@@ -0,0 +1,56 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.microsoft.aad.msal4j;
+
+import java.net.MalformedURLException;
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.net.URL;
+import java.util.HashMap;
+import java.util.Map;
+
+class ManagedIdentityRequest extends MsalRequest {
+
+    URI baseEndpoint;
+
+    HttpMethod method;
+
+    Map<String, String> headers;
+
+    Map<String, String> bodyParameters;
```

Comment:
headers and body parameters are not used.

Reply:
Headers is used in IMDS and AppService. The bodyParameters will be used in one of the sources that have POST method. So added it in advance.